packages/donate-button-beta/src/components/widget/types/widget-config.ts

```diff
@@ -16,11 +16,11 @@ export type WidgetConfig = {
 	countrySelection: boolean;
 	primaryColor: string;
 	forceLanguage: string | false;
-	defaultDonationAmount: number;
+	defaultDonationAmount?: number;
 	currencies: CurrencyOption[];
 	defaultFrequency: DonationFrequency;
 	showInitialMessage: boolean;
 	countries: DonationRecipient[];
 	infoPages: InfoPage[];
-	i18n: Record<string, Language>; // ISO Alpha-2 format
+	i18n: Record<string, Partial<Language>>; // ISO Alpha-2 format
```

Comment:
Yea, ideally this would be more like this:

```suggestion
	i18n: Record<"en", Language> & Record<Exclude<string, "en">, Partial<Language>>; // ISO Alpha-2 format
```